pvlib/shading.py

```diff
@@ -553,3 +553,140 @@ def shaded_fraction1d(
     )
 
     return np.clip(t_asterisk, 0, 1)
+
+
+def direct_martinez(
+    poa_global,
+    poa_direct,
+    shaded_fraction,
+    shaded_blocks,
+    total_blocks,
+):
+    r"""
+    A shading losses power factor for non-monolithic Silicon
+    modules and arrays with an arbitrary number of bypass diodes.
+
+    This experimental model cancels a fraction of the direct and circumsolar
```

Comment:
I don't understand what "cancels" means in this context

Reply:
I've changed the verb to "zeroes":

`     This experimental model zeroes a fraction of the direct and circumsolar`

I may have not translated that correctly. I've seen "to zero" as a verb. Should I reword it to something along `This experimental model decreases a fraction of the direct and circumsolar with each shaded block`

Reply:
Let's use the word reduces

> This experimental model reduces a fraction of the direct and circumsolar with each shaded block